src/zzz_od/application/world_patrol/world_patrol_app.py

```
@@ -105,41 +107,51 @@ def back_after_stop_tracking(self) -> OperationRoundResult:
+        run_record = cast(WorldPatrolRunRecord, self.run_record)
+        if route.full_id in run_record.finished:
             self.route_idx += 1
             return self.round_wait(status=f'跳过已完成路线 {route.full_id}')
 
         op = WorldPatrolRunRoute(self.ctx, route)
         result = op.execute()
 
-        # 特殊处理：卡住脱困超过上限时，等待3秒并从当前路线起点重启一次
-        if not result.success and isinstance(result.status, str) and '卡住超限，重启当前路线' in result.status:
+        def _is_stuck_over_limit_status(status: object) -> bool:
+            return isinstance(status, str) and '卡住超限，重启当前路线' in status
```

Ruff (0.14.3)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)